arangod/GeneralServer/IoContext.cpp

```diff
@@ -25,6 +25,11 @@
 
 #include <thread>
 
+#include "Basics/cpu-relax.h"
```

Comment:
I don't think we need to include "cpu-relax.h" here.
While waiting, the code here calls `std::this_thread::yield()` and nothing from cpu-relax.h.
Or am I missing something?

Reply:
Done